extensions/ql-vscode/src/pure/disposable-object.ts

```diff
@@ -1,4 +1,11 @@
-import { Disposable } from 'vscode';
+
+// Avoid explicitly referencing Disposable type in vscode.
+// This file cannot have dependencies on the vscode API.
+interface Disposable {
+  dispose(): any;
```

Comment:
Is `unknown` better than `any` here.

Reply:
The vscode api uses `any`. I was trying to copy that.